plugins/builds.js

```diff
@@ -87,11 +118,32 @@ exports.plugin = {
                     }
                 }
 
+                const fileName = artifact.split('/').pop();
+                const fileExt = fileName.split('.').pop();
+                let mime = getMimeFromFileExtension(fileExt);
+
+                if (value && !value.h && value.type === 'Buffer') {
```

Comment:
guess we don't need it here since the only type we care about is html

Reply:
Removed